src/PanoImageRenderer/PanoImageRenderer.js

```diff
@@ -38,7 +39,7 @@ const ERROR_TYPE = {
 };
 
 export default class PanoImageRenderer extends Component {
-	constructor(image, width, height, sphericalConfig) {
```

Comment:
`isVideo` is used to distinguish `image` is image or video.

Other consideration
  1. image => content
     - Issue: It's difficult to distinguish image from image if content is `string`.
  2. Add isVideo property on sphericalConfig
     - Issue: isVideo is not related with sphericalConfig

Reply:
Understood.